example/shchehelskyiho/second/task1.py

```diff
@@ -0,0 +1,32 @@
+'''
+Користувач вводить час у форматі `hh:mm(:ss(.ms))`, де дужки позначають необов'язковість.
+Напишіть функцію, що повертає кількість повних секунд, що пройшли з опівночі.
+Виведіть результат.
+У випадку помилкового вводу виведіть повідомлення про помилку.
+'''
+import re
```

Comment:
Can be replaced with `from re import match` to save such important nanoseconds or maybe more( sometimes it really helps ).